website/docs/docs/cloud/about-cloud/architecture.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "dbt Cloud Architecture"
+id: "architecture"
+description: "Information about the architecture, communication, and security of dbt Cloud"
+---
+
+This page is intended to help practitioners and those interested in the architecture and data flow of the hosted dbt Cloud product.
+
+## About dbt Cloud architecture
+
+The dbt Cloud application comprises a set of static and dynamic components. The static components are constantly running to serve highly available dbt Cloud functionality, for example, the dbt Cloud web application. The dynamic components are created ad-hoc to fill background jobs or requests to use the IDE. 
+
+dbt Cloud is available in most regions around the world in both [single tenant](/docs/cloud/about-cloud/tenancy#single-tenant)(AWS and Azure) and [multi-tenant](/docs/cloud/about-cloud/tenancy#multi-tenant) configurations.  
+
+dbt Cloud uses PostgreSQL for its backend, S3-compatible Object Storage systems for logs and artifacts, and a Kubernetes storage solution for creating dynamic, persistent volumes. 
+
+All data at rest on dbt Cloud servers is protected using AES-256 encryption. 
+
+<img src="/img/docs/dbt-cloud/on-premises/data-flows.png" />
+
+For a more detailed breakdown of the dbt Cloud apps, [download the advanced architecture guide PDF](https://drive.google.com/uc?export=download&id=1lktNuMZybXfqFtr24J8zAssEfoL9r51S).
+
+## Communication
+
+dbt Cloud can communicate with several external services, including data platforms, git repositories, authentication services, and directories. All communications occur over HTTPS (attempts to connect via HTTP are redirected to HTTPS). dbt Cloud encrypts in transit using the TLS 1.2 cryptographic protocol. 
+
+TLS (Transport Layer Security) 1.2 is an industry-standard protocol for encrypting sensitive data while it travels over the public internet (which does not offer native encryption).
+
+A typical scenario that might be seen frequently is an employee working in a public space, such as an airport or café. The user might be connected to an unsecured public network offered by a facility to which many others are also connected. What if there is a bad actor amongst them running a program (such as Wireshark) that can "capture" network packets and analyze them over the air?
```

Comment:
great example! only small question of whether we want to explicitly use the name 'wireshark' and associate with bad actor? not sure if I'm being overly cautious though!

Reply:
Good call out, but yes, Wireshark is a tool common to professionals and hackers alike. It's the most common tool used to monitor internet traffic over the air, capture network packets, and read unencrypted traffic. They're well aware of their status as a tool used for both good and evil.